contracts/src/v0.8/vrf/test/VRFV2Plus.t.sol

```diff
@@ -1,19 +1,19 @@
 pragma solidity 0.8.19;
 
-import "../BaseTest.t.sol";
-import {VRF} from "../../../../src/v0.8/vrf/VRF.sol";
-import {MockLinkToken} from "../../../../src/v0.8/mocks/MockLinkToken.sol";
-import {MockV3Aggregator} from "../../../../src/v0.8/tests/MockV3Aggregator.sol";
-import {ExposedVRFCoordinatorV2_5} from "../../../../src/v0.8/vrf/dev/testhelpers/ExposedVRFCoordinatorV2_5.sol";
-import {VRFCoordinatorV2_5} from "../../../../src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol";
-import {SubscriptionAPI} from "../../../../src/v0.8/vrf/dev/SubscriptionAPI.sol";
-import {BlockhashStore} from "../../../../src/v0.8/vrf/dev/BlockhashStore.sol";
-import {VRFV2PlusConsumerExample} from "../../../../src/v0.8/vrf/dev/testhelpers/VRFV2PlusConsumerExample.sol";
-import {VRFV2PlusClient} from "../../../../src/v0.8/vrf/dev/libraries/VRFV2PlusClient.sol";
-import {VRFTypes} from "../../../../src/v0.8/vrf/VRFTypes.sol";
+import "./BaseTest.t.sol";
```

Comment:
Should we update any CI checks for this move?

Reply:
CI uses foundry.toml to run tests, that's why the test location has been updated in that file